iib/workers/tasks/build.py

```diff
@@ -687,6 +696,11 @@ def inspect_related_images(bundles: List[str], request_id) -> None:
             bundle_metadata = get_bundle_metadata(operator_manifest, False)
             for related_image in bundle_metadata['found_pullspecs']:
                 related_image_pull_spec = related_image.to_str()
+                if replace_related_image_registry:
+                    related_image_regsitry = related_image_pull_spec.split('/')[0]
```

Comment:
Maybe it's a good idea to log that you're replacing the registry here?

Reply:
Sure, I don't mind adding another log message.
The image pull spec is already mentioned in the skopeo inspect  log, so I thought it wouldn't be necessary

Reply:
Added the log message

Reply:
A log message might be more helpful for debugging purposes